app/adapters/application.js

```diff
@@ -11,4 +11,12 @@ export default class ApplicationAdapter extends RESTAdapter {
   headers = {
     Authorization: this.session.token ? `Token ${this.session.token}` : '',
   };
+
+  handleResponse(status, headers, payload) {
```

Comment:
We need to convert REST errors to JSON:API format. Even though the [Ember Data docs mention we don't need to explicitly convert errors](https://api.emberjs.com/ember-data/release/classes/DS.RESTAdapter), it appears we do. To do the conversion I use the RESTAdapter's `errorsHashToArray` method which translates the following:

```js
// BEFORE
let errors = {
  name: 'Must be present',
};

// AFTER
  [
    {
      title: "Invalid Attribute",
      detail: "Must be present",
      source: { pointer: "/data/attributes/name" }
    },
  ]
```